### PR TITLE
updated the sponsor section's hover and opacity

### PR DESCRIPTION
**Pull request by athul**
some usual shits....

### CODE REVIEW

**Comment by haxzie-xx** [comment]
@Athul-CA check the sponsors section on mobile, a few pixels of vertical padding between them would make it look good and uncluttered.

**Comment by athul** [comment]
> @Athul-CA check the sponsors section on mobile, a few pixels of vertical padding between them would make it look good and uncluttered.

@haxzie Fixed it.

File: css/main.css [hunk 1]
```diff
@@ -735,15 +735,17 @@ select.error {
...
 }
 .block-sponsor img:hover {
-    opacity: .4;
-    transform: scale(1.05);
-    -webkit-transform: scale(1.05);
-    -ms-transform: scale(1.05);
-    -moz-transform: scale(1.05);
+    padding-top: 25px;
+    opacity: 1.10;
+    transform: scale(1.50);
```
Comment (haxzie-xx) [inline css/main.css]: The scale ratio is too much, decrease it for a smooth experience. A tiny amount will do the magic.

**Review by haxzie-xx — CHANGES_REQUESTED** [review]

File: css/main.css (hunk 1 above)
Comment (athul) [reply css/main.css]: changed the transform scale to 1.10(kept at 1 for emphasis animations)

**Comment by athul** [comment]
I Think id bf9b66c0048363b659e7d66e0b4078102c49e7c9 will resolve #11

**Comment by haxzie-xx** [comment]
Could you try some other gradient for the button? 🧙🏽‍♂️

**Comment by athul** [comment]
Any colour suggestions? I'll try em

**Comment by haxzie-xx** [comment]
Anything that goes along with the color of the logo 😉 You can find em from https://grabient.com or https://uigradients.com or https://webgradients.com

**Comment by haxzie-xx** [comment]
Is the sponsors section missing?

**Comment by athul** [comment]
No... Ican see t in the deploy preview

**Comment by haxzie-xx** [comment]
Screenshot from 2019-03-29 23-03-56

**Comment by athul** [comment]
image

I am getting em

**Pull request merged**